enterprise/server/usage/usage_config/usage_config.go

```diff
@@ -0,0 +1,9 @@
+package usage_config
+
+import "flag"
+
+var usageTrackingEnabled = flag.Bool("app.usage_tracking_enabled", false, "If set, enable usage data collection.")
+
+func UsageTrackingEnabled() bool {
+	return *usageTrackingEnabled
+}
```

Comment:
would it work to move this to `usage.go` and have `usage_service` depend on `usage.go` instead? Would be nice to remove this extra pkg if possible

Reply:
We could, but it would create an otherwise unnecessary dependency on `usage.go`, which could mean that `app.region` could end up defined for a tool that only makes use of `usage_service`, which would make it a meaningless flag. I tried to avoid this as a design goal with all of these changes, but if we feel that I'm being over-cautious, I can switch it here.

Reply:
Gotcha, I hadn't considered the effects of pulling in unused flags as deps. sgtm